src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/RedirectsServiceTests.cs

```diff
@@ -0,0 +1,403 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Utils.ContentDbUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Tests
+{
+    public class RedirectsServiceTests
+    {
+        [Fact]
+        public async Task List_MethodologyRedirect()
+        {
+            var latestPublishedVersionId = Guid.NewGuid();
+            var methodology = new Methodology
```

Comment:
Might be good to get some practice in using the DataFixtures soon.  I don't mind if that's the *next* PR though :)

Reply:
I'll use it next PR. I've been mostly avoiding it cause I feel like it'll take me a while to get my head round it, and I'd prefer to remain productive. But I'm going to have to learn it at some point.